crnk-gen/crnk-gen-asciidoc/src/main/java/io/crnk/gen/asciidoc/capture/AsciidocCaptureModule.java

```diff
@@ -95,15 +90,32 @@ public Object decorateRepository(Object repository) {
 			if (repository instanceof ResourceRepository) {
 				return new AsciidocResourceDecorator((ResourceRepository) repository, this);
 			}
+			if (repository instanceof RelationshipRepository) {
```

Comment:
looks great like this! 

One small last thing  that we can merge: can you change `repository instanceOf RelationshipRepository` to `repository instanceof OneRelationshipRepository && repository instanceof ManYRelationshipRepository`?

Reply:
because the old RelationshipRepository is deprecated, but users may still continue to implement both one and many in the same class.

Reply:
Ok! In several hours I will make the changes.